Source/Parallelization/WarpXComm.cpp

```diff
@@ -1303,6 +1322,24 @@ void WarpX::NodalSyncPML (int lev, PatchType patch_type)
             WarpXCommUtil::OverrideSync(*pml_G, period);
         }
     }
+
+#ifdef WARPX_DIM_RZ
+    if (pml_rz[lev])
+    {
+        // This is not actually needed with RZ PSATD since the
+        // arrays are always cell centered. Keep for now since
+        // it may be useful if the PML is used with FDTD
+        const auto& pml_rz_E = pml_rz[lev]->GetE_fp();
+        const auto& pml_rz_B = pml_rz[lev]->GetB_fp();
+
+        // Always synchronize nodal points
+        const auto& period = Geom(lev).periodicity();
```

Comment:
```suggestion
        const amrex::Periodicity& period = Geom(lev).periodicity();
```

Reply:
Fixed. BTW, `auto` is used in many other places in this files. Maybe a clean up PR can be submitted?

Reply:
Thanks, totally, I can open a cleanup PR to do this.

Reply:
Done in #2578.